private-templates-service/client/package.json

```diff
@@ -5,6 +5,13 @@
 	"dependencies": {
 		"@fortawesome/fontawesome-free": "^5.11.2",
 		"@microsoft/microsoft-graph-client": "^2.0.0",
+		"@types/jest": "^24.9.1",
```

Comment:
It'd be better to put dependencies to dev dependencies as much as possible, to keep your bundle size smaller. 
Jest is not needed to run your code in prod.